Formula/julia.rb

```diff
@@ -104,8 +73,6 @@ def install
       prefix=#{prefix}
       sysconfdir=#{etc}
       USE_SYSTEM_CSL=1
-      USE_SYSTEM_LLVM=1
-      USE_SYSTEM_LIBUNWIND=1
```

Comment:
We may want to keep these lines as
```sh
       USE_SYSTEM_LLVM=0
       USE_SYSTEM_LIBUNWIND=0
```
to make intent clearer.

Reply:
The formula is long enough already, I think